app/controllers/declaration/CarrierDetailsPageController.scala

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.declaration
+
+import config.AppConfig
+import controllers.actions.{AuthAction, JourneyAction}
+import controllers.util.CacheIdGenerator.cacheId
+import forms.declaration.CarrierDetails
+import javax.inject.Inject
+import play.api.data.Form
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.mvc.{Action, AnyContent}
+import services.CustomsCacheService
+import uk.gov.hmrc.play.bootstrap.controller.FrontendController
+import views.html.declaration.carrier_details
+
+import scala.concurrent.{ExecutionContext, Future}
+
+/**
+  * This controller is not used in supp dec journey
+  */
+class CarrierDetailsPageController @Inject()(
+  appConfig: AppConfig,
```

Comment:
Could you, please, make the appConfig an implicit?

Reply:
:man_facepalming: Forgot about that!!!!!